vignettes/esqlabsR.Rmd

```diff
@@ -0,0 +1,212 @@
+<!-- Data importer configuration: Data/esqlabs_dataImporter_configuration.xml  -->
+<!-- Compound Properties File: Data/Compound Properties.xlsx  -->
+<!-- Output folder: Results  -->
+
+Notice that the `ProjectConfiguration` links to the other and files that 
+constitute the project.
+
+For more information about esqlabsR's project structure, read `vignette("esqlabsR-project-structure")`.
+
+## Design Simulations
+
+Now that our project is ready, the next step is to run simulations. In esqlabsR,
+simulation are run by defining and executing multiple **scenarios**.
+
+To modify, add or delete a scenario, we edit the `Parameters/Scenarios.xslx` 
+file. Each row of this file is a scenario. Each Column is a parameter that will 
+be used to define the scenarios.
+
+Here are the main parameters that can be set for a scenario:
+
+- `ModelFile` is the name of the `.pkml` file that can be found in the `Models/` directory.
+- `ModelParameterSheets` is the name of the *sheet* in `Parameters/ModelParameters.xlsx` that contains the parameters to apply to the model,
+- `ApplicationProtocol` is the name of the *sheet* in `Parameters/ApplicationParameters.xlsx` that specify the simulated drug administration protocol,
+- `SimulationTime` and `SimulationTimeUnit` define the time range to simulate, it should be three numbers for start, end, resolution
+- `Individuald`: the id of the individual to simulate. The id and settings for individuals are defined in `Parameters/Individuals.xlsx`,
+- `PopulationId`: the id of the *population* to simulate. The id and settings for populations are defined in `Parameters/PopulationParameters.xlsx`,
+
+Notice that the `Scenarios.xslx` file is mainly constituted of "link" to other excel templates. 
+
+![](Figures/scenarios_links_files.png)
+
+For example, if we want to change the characteristic of the individual we run the simulation on, we can either edit its properties directly in the `Individuals.xlsx` file or  add a new row with a different `IndividualId` and change to this new value in the `Scenarios.xslx` file.
+
+For more information on all the scenario parameters that can be setup, read `vignette("esqlabsR-design-simulations")`.
+
+
+## Run Simulations
+
+Once all the scenario parameters are all setup in th excel files, we can import 
+them using `createScenarios` and `readScenarioConfigurationFromExcel`:
+
+```{r, echo=FALSE, warning=FALSE, message = FALSE}
+my_project_configuration <- createDefaultProjectConfiguration(path = example_ProjectConfiguration())
+```
+
+```{r}
+my_scenarios <-  createScenarios(
+  readScenarioConfigurationFromExcel( 
+    scenarioNames = "TestScenario", 
+    projectConfiguration = my_project_configuration))
+```
+
+```{r, echo = FALSE}
+my_scenarios$TestScenario$scenarioConfiguration$print(projectConfiguration = FALSE)
+```
+
+Then, we run the simulations by passing the scenarios we defined:
+```{r}
+my_simulation <- runScenarios(my_scenarios)
```

Comment:
Rename the variable to `myScenarioResults` so there is not confusion between the `Simulation` object from `{ospsuite}` and what we are having here as results.